bitnami/airflow/README.md

```diff
@@ -19,7 +19,7 @@ Bitnami charts can be used with [Kubeapps](https://kubeapps.com/) for deployment
 - Kubernetes 1.12+
```

Comment:
Please, take a look at the _README.md_ there are several code blocks out of place and some weird characters.
If you are not adding any content, just updating the table with the new parameters you should use [the `readme-generator` tool](https://github.com/bitnami-labs/readme-generator-for-helm) in order to generate fill the tables that appear in the _README.md_ using the metadata from the _values.yaml_

Reply:
working on that, seems I have some problems with the tool. I'll be back as soon they are fixed.

Reply:
fixed1